addon/mixins/route.js

```diff
@@ -174,7 +174,8 @@ export default Ember.Mixin.create({
     }
 
     var params = Ember.merge(requestPayloadBase, options);
-    let promise = this.store[this._storeFindMethod](modelName, params);
+
+    let promise = this.afterInfinityModel(this.store[this._storeFindMethod](modelName, params));
```

Comment:
Ha, there was another implementation of this that I tried out and didn't commit:

```javascript
let promise = this.store[this._storeFindMethod](modelName, params).then(afterInfinityModel);
```

Something like that?